test/htmlnano_compat.cjs

```diff
@@ -14,6 +14,8 @@ describe('[commonjs usage]', () => {
     });
 
     it('PostHTML plugin', () => {
+        console.log({ htmlnano })
```

Comment:
Should it be removed?

Reply:
Ah yes! This was debugging the ESM-CJS interop and I just forgot to remove!

Reply:
Done in [`59b5dad` (#339)](https://github.com/posthtml/htmlnano/pull/339/commits/59b5dad4535f44eb5523d8b4fc3824f0deb39da9)